roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -67,20 +74,25 @@ void loadComputeLibrary() {
   }
 #endif
 
-#endif
 
   for (auto&& libName : libNames) {
     libName += libSuffix;
     const auto returnValue = gSystem->Load(libName.c_str());
 
     if (returnValue == -1 || returnValue == -2) {
-      throw std::runtime_error("RooFit was unable to load its computation library " + libName);
+      std::cout << "RooFit was unable to load its computation library " + libName << std::endl;
     }
     // Library should not have been loaded before we tried to do it.
-    if (returnValue == 1) {
+    else if (returnValue == 1) {
```

Comment:
It would be better to contract those lines:
```
} else if (returnValue == 1) {
  // Library should not ...
```
The RooFit code usually has the `else if` on the same line.